lib/spacesuit/auth_middleware.ex

```diff
@@ -12,17 +12,7 @@ defmodule Spacesuit.AuthMiddleware do
         {:ok, req, env}
 
       "Bearer " <> token ->
```

Comment:
As a general rule, when I see a nested case/switching, I look to see if there is a way to avoid having to keep the branching logic in my head. Elixir typically gives us enough options that we can avoid it without making the code more complicated.